src/textual/app.py

```diff
@@ -369,6 +369,11 @@ def __init__(
         self.css_path = css_paths
         self._registry: WeakSet[DOMNode] = WeakSet()
 
+        self.scroll_sensitivity_x: float = 4
```

Comment:
Scrolling in x dimension is double to make the perceived distance the same in both directions (given cells are typically twice as tall as they are wide).

Reply:
Isn't that a good `# comment` in the original source?